themes/medius/scss/components/links.scss

```
@@ -0,0 +1,235 @@
+  &:focus {
+    outline: none;
+
+    &:after {
+      content: "";
+      position: absolute;
+      display: block;
+      left: 0;
+      top: 0;
+      right: 0px;
+      bottom: 0;
+      border-radius: 4px;
+      box-shadow: 0 0 0 2px $coral-60 inset, 0 0 0 3px $yellow-10 inset;
+    }
+  }
+
+  @media (min-width: $tablets) {
+    font-size: 20px;
+    line-height: 1.4;
+    padding: 4px 12px 4px 4px;
+  }
+}
+
+.link-swipe-right {
+  @include ff-sans($weight-bold);
+
+  font-size: 16px;
+  line-height: 2;
+  display: inline-block;
+  position: relative;
+  padding-left: 12px;
+  padding-right: 12px;
+  text-decoration: none;
+  transition: all 0.2s ease;
+  z-index: 0;
+  color: $yellow-10;
+  vertical-align: middle;
+
+  &:before {
+    content: "";
+    height: 100%;
+    width: 4px;
+    position: absolute;
+    display: block;
+    right: 8px;
+    bottom: 0;
+    left: 0;
+    z-index: -1;
+    transition: all 0.2s ease;
+    border-radius: 4px;
+    background-color: $coral-60;
+  }
+
+  &:hover,
+  &:focus {
+    color: $yellow-10;
+    padding-left: 16px;
+    padding-right: 8px;
+    border-radius: 4px;
+    text-decoration: none;
+
+    &:before {
+      height: 100%;
+      right: 0;
+      left: 0;
+      width: 100%;
+    }
+  }
+
+  &:focus {
+    outline: none;
+
+    &:after {
+      content: "";
+      position: absolute;
+      display: block;
+      left: 0;
+      top: 0;
+      right: 0px;
+      bottom: 0;
+      border-radius: 4px;
+    }
+  }
+}
+
+.link--white {
+  color: $white-pure;
+
+  &:before {
+    background-color: $white-pure;
+  }
+
+  &:hover,
+  &:focus {
+    color: $purple-60;
+  }
+  &:focus:after {
+    box-shadow: 0px 0px 0 2px $white-pure inset, 0 0px 0 3px $black-50 inset;
+  }
+}
+
+.link--yellow-30 {
+  color: $white;
+
+  &:before {
+    background-color: $yellow-30;
+  }
+
+  &:hover,
+  &:focus {
+    color: $black-87;
+  }
+  &:focus:after {
+    box-shadow: 0px 0px 0 2px $yellow-30 inset, 0 0px 0 3px $black-50 inset;
+  }
+  @media (min-width: $desktops) {
+    color: $black-87;
+  }
+}
+
+.link--teal-30 {
+  color: $white;
+
+  &:before {
+    background-color: $teal-30;
+  }
+
+  &:hover,
+  &:focus {
+    color: $black-87;
+  }
+  &:focus:after {
+    box-shadow: 0px 0px 0 2px $teal-30 inset, 0 0px 0 3px $black-50 inset;
+  }
+  @media (min-width: $desktops) {
+    color: $black-87;
+  }
+}
+
+.link--coral-30 {
+  color: $white;
+
+  &:before {
+    background-color: $coral-30;
+  }
+
+  &:hover,
+  &:focus {
+    color: $black-87;
+  }
+  &:focus:after {
+    box-shadow: 0px 0px 0 2px $coral-30 inset, 0 0px 0 3px $black-50 inset;
+  }
+  @media (min-width: $desktops) {
+    color: $black-87;
+  }
+}
+
+.link--purple-30 {
+  color: $white;
+
+  &:before {
+    background-color: $purple-30;
+  }
+
+  &:hover,
+  &:focus {
+    color: $black-87;
+  }
+  &:focus:after {
+    box-shadow: 0px 0px 0 2px $purple-30 inset, 0 0px 0 3px $black-50 inset;
+  }
+  @media (min-width: $desktops) {
```

```suggestion
  @media (min-width: $large-desktops) {
```

Also you might create a new variable named `$tablet-landscape`